### ISSUE TITLE
Bug with Steam Review system, creates bugged/frozen dropdown menu following all screens when selecting 'Display As'

### ISSUE THREAD

**Issue by NLBen** [ValveSoftware/steam-runtime#461]
#### Your system information

* Steam Runtime Version: 
20 Juli 2021 22:25, API v020, package version 1626824053

* Distribution (e.g. Ubuntu 18.04): 
Linux Mint 20.2 Uma Xfce

* Link to your full system information
https://gist.github.com/NLBen/8a35467293bcbc745513aaf294b3df6b

* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
https://gist.github.com/NLBen/6f4cdb2636280471c0e4286ed668f452
This is the result of me starting Steam through the LOG=1 command. What I did there is besides opening Steam going to a store page and 'ticking off' the relevant bug as well as requesting the Steam data (for Steam runtime info) as well as request the System Information.

Basically what happens when it comes to the bug is that I:
- Open Steam
- Select an game I have an interest in and check the review system
- Want to further customize how I see some things by using Display As
- And then notice that the Display As drop-down menu freezes and start following me if I tab over to Firefox, my desktop or anywhere else (including if I change to somewhere else within Steam itself, either the Library or another Steam Store page).
- Which seemingly can only be solved by shutting Steam down

#### Steps for reproducing this issue:

1.  Open Steam in Linux
2. Go to the review section of any game
3. Select the 'toggle' from Display As

**Comment by kisak-valve** [comment]
Hello @NLBen, this issue is already being tracked at https://github.com/ValveSoftware/steam-for-linux/issues/7825. Closing as a duplicate.
